css/index.css

```diff
@@ -1,3 +1,331 @@
+  margin: 0 auto;
+
+}
+
+
+
+/* header */
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+header nav {
+	display: flex;
+  flex-flow: row flex-start;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 20px;
+  width: 70%
+}
+
+header nav a {
+  display: flex;
+  flex-flow: row wrap;
+  font-size: 16px;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  /*margin-left: 0;*/
+}
+
+header nav a:last-child {
+  /*margin-right: 0;*/
+}
+
+header .logo {
+  display: flex;
+  flex-flow: row-reverse;
+
+ }
+
+/* cta */
+.cta {
+  display: flex;
+  margin-top: 50px;
+  flex-flow: row;
+  justify-content: space-around;
+}
+
+.cta .cta-text {
+  display: flex;
+  flex-flow: column;
+  font-size: 77px;
+  text-align: center;
+  align-items: center;
+	align-self: center;
+
+}
+
+.cta .cta-text button {
+  display: flex;
+  justify-content: center;
+  background: white;
+  margin: 10px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+  width: 50%;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display: flex;
+  flex-flow: row-reverse;
+
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  display: flex;
+  padding: 0 10px;
+}
+
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  display: flex;
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+  display: flex;
+  margin-top: 37px;
+  padding: 0 10px;
+}
+
+.main-content .bottom-content .text-content {
+
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+/* Tablet view */
+@media (max-width: 800px) {
+  .container {
+		width: 100%;
+	}
+	header {
+    display: flex;
+    flex-direction: column-reverse;
+		justify-content: center;
+		align-content: center;
+		align-items: center;
+		margin-bottom: 30px;
+		width: 100%;
+	}
+	header .logo {
+		display: flex;
+    width: 30%;
+		justify-content: center;
+		margin-top: 30px;
+		margin-bottom: 30px;
+	}
+	header nav {
+		align-items: flex-end;
+		width: 100%;
+		justify-content: space-around;
+	}
+	section .cta {
+   display: flex;
+	 justify-content: center;
+	 margin-top: 20px;
+	 margin-bottom: 20px;
+	}
+	 .cta img {
+		 display: none;
+	 }
+	 .cta p {
+		 padding: 5px;
+	 }
+	 .cta-text {
+		 padding-bottom: 30px;
+	 }
+	 .main-content {
+		 margin: 20px;
+	 }
+	 .contact {
+		 margin: 20px;
+	 }
+	 .middle-img {
+		 display: flex;
+		 align-content: center;
+		 justify-content: center;
+		 width: 100%;
+		 }
+	}
+
+
+/* Mobile view */
+@media (max-width: 500px) {
```

Comment:
Mobile View Notes
- [ ] The logo looks a bit larger and more centered in the design file. You may need to change the width and spacing in your 500px media query.
- [ ] Play with the font-size.
- [ ] The "Innovation On Demand" text could be smaller in the mobile view.